src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -260,6 +260,20 @@ kibana_vars=(
     xpack.rollup.enabled
     xpack.searchprofiler.enabled
     xpack.security.audit.enabled
+    xpack.security.audit.appender.type
+    xpack.security.audit.appender.layout.type
+    xpack.security.audit.appender.layout.highlight
+    xpack.security.audit.appender.layout.pattern
+    xpack.security.audit.appender.legacyLoggingConfig
+    xpack.security.audit.appender.fileName
+    xpack.security.audit.appender.policy.type
+    xpack.security.audit.appender.policy.interval
+    xpack.security.audit.appender.policy.modulate
+    xpack.security.audit.appender.policy.size
+    xpack.security.audit.appender.strategy.type
+    xpack.security.audit.appender.strategy.max
+    xpack.security.audit.appender.strategy.pattern
```

Comment:
Many of these keys changed in 7.12 via #90764 -- `kind` -> `type`, and `path` -> `fileName`.

Since we don't have anyone using these values in Docker yet, I think we should only support the new values, and we should not backport this PR to 7.11.